server/middlewares/paper-trail.js

```
@@ -1,45 +1,190 @@
-      await paperTrailService.createPaperTrail(
-        ctx,
-        schema,
-        uid,
-        change,
-        isAdmin
-      );
+        // Get the current user from async storage
+        console.log('[Paper Trail Debug] About to call getCurrentUser()');
+        const currentUser = getCurrentUser();
+        console.log('[Paper Trail Debug] Current user from AsyncLocalStorage:', currentUser);
+        console.log(`Paper Trail middleware detected user:`, currentUser);
```

ESLint

[error] Strings must use singlequote.

(quotes)